src/tag.rs

```diff
@@ -26,23 +26,46 @@ use crate::decode;
 ///
 /// # Limitations
 ///
-/// At this time, we can only decode single-octet identifier octets. That is,
-/// we only support tag numbers between 0 and 30.
+/// At this time, we can only decode up to four identifier octets. That is,
```

Comment:
Given that later on we return the tag number as an `u32`, perhaps we should state this as a permanent limitation?

Reply:
Ok. Removed "At this time, ".